en/docs/streaming/connectors/downloading-and-installing-siddhi-extensions.md

```diff
@@ -0,0 +1,174 @@
+# Downloading and Installing Siddhi Extensions
+
+The Siddhi extensions supported for the Streaming Integrator are shipped with the product by
+default. If you need to download and install a different version of an
+extension, you can download it via the command line or manually as covered in the following sections.
+
+## Downloading and installing Siddhi extensions via the command line
+
+To manage Siddhi extensions via the command line, see the following topics.
+
+### Identifying the Siddhi extensions to install/uninstall
+
+The following are some actions that you are required to perform in order to identify the Siddhi extensions you need to install. Navigate to the `<SI_HOME>/bin` directory in the CLI to issue these commands.
+
+- **Viewing the list of extensions that are currently installed**
+
+    You can view the complete list of Siddhi extensions that are currently installed in your Streaming Integrator setup. All the extensions listed are completely installed with the dependencies.
+    
+    To perform this action, issue the appropriate command out of the following based on your operating system:
+    
+    - **For Windows**     : `extension-installer.bat list`
+    - **For Linux/MacOS** : `./extension-installer.sh list`
+    
+    The following is a sample response log for this command.
+    
+    ![List of Installed Extensions](../images/downloading-and-installing-siddhi-extensions/list-response.png)
```

Comment:
Image path is wrong as there isnt a folder named "images" in the APIM repo. This issue has been repeated in other locations as well.